lib/go-qmstr/reporting/package_data.go

```diff
@@ -12,6 +13,9 @@ type PackageData struct {
 	LicenseDeclared string
 	Targets         []*Target
 	Project         string // The project this package is associated with.
+	CleanedProject  string // Project name without slashes
```

Comment:
Instead of creating new fields, you can directly use the RemoveSlash() while creating the package data.